--- alsa-tools-1.0.11.orig/hdspconf/configure.in	2004-03-19 16:31:13.000000000 +0100
+++ alsa-tools-1.0.11/hdspconf/configure.in	2006-08-23 22:49:46.000000000 +0200
@@ -36,6 +36,6 @@
 FLTK_LIBS=`$FLTK_CONFIG $fltk_config_args --ldflags`
 
 CXXFLAGS="$CXXFLAGS $ALSA_CFLAGS $FLTK_CXXFLAGS"
-LDFLAGS="$LDFLAGS $ALSA_LIBS $FLTK_LIBS"
+LIBS="$LIBS $ALSA_LIBS $FLTK_LIBS"
 
 AC_OUTPUT(Makefile src/Makefile pixmaps/Makefile)
