diff -ur alsa-tools-1.0.27.org/ld10k1/src/ld10k1.c alsa-tools-1.0.27/ld10k1/src/ld10k1.c
--- alsa-tools-1.0.27.org/ld10k1/src/ld10k1.c	2013-04-11 13:43:07.000000000 +0200
+++ alsa-tools-1.0.27/ld10k1/src/ld10k1.c	2013-04-14 21:18:25.553846590 +0200
@@ -62,7 +62,7 @@
 			 localtime(&tp));
 		fprintf(out, "%s %s", timestr, label);
 	} else 
-		fprintf(out, label);
+		fprintf(out, "%s", label);
 	vfprintf(out, fmt, va);
 	fprintf(out, "\n");
 	fflush(out);
diff -ur alsa-tools-1.0.27.org/ld10k1/src/lo10k1.c alsa-tools-1.0.27/ld10k1/src/lo10k1.c
--- alsa-tools-1.0.27.org/ld10k1/src/lo10k1.c	2013-04-11 13:43:07.000000000 +0200
+++ alsa-tools-1.0.27/ld10k1/src/lo10k1.c	2013-04-14 21:12:42.462505232 +0200
@@ -868,7 +868,7 @@
 
 void debug_print(char *str)
 {
-	printf(str);
+	printf("%s", str);
 }
 
 static int debug(int deb)
