--- alsa-tools-1.0.8/as10k1/as10k1.c.orig	2003-07-15 20:33:59.000000000 +0200
+++ alsa-tools-1.0.8/as10k1/as10k1.c	2005-05-03 23:17:51.000000000 +0200
@@ -93,7 +93,7 @@
 					dbg_opt=dbg_opt|DBG_INSTR;//default
 					i--;
 					printf(" instructions by default\n");
-					goto next_cli_arg;
+					continue;
 				}
 				
 				while(argv[i][j]!='\0'){
@@ -158,8 +158,6 @@
 			}
 
 		}
-next_cli_arg:
-
 	}
 }
 
